### ISSUE TITLE
Integrity Checker client needs to use cookies 

### ISSUE THREAD

**Issue by wezell** [dotCMS/core#18554]
If you run the integrity checker against a dotCMS cluster - running behind a sticky session load balancer, it fails.  This is because the integrity checker uses the Jersey Client to makes multiple requests to generate the integrity data and also to check if that generation is done, etc..  The issue is that our use of the Jersey Client does not accept cookies and our integrity check and data is not clustered.  This means that these multiple requests hit different servers in the cluster and ultimately fail.

Also, the integrity checker needs to respect the auth token like the push publishing endpoint does.

Also, in the RestClient, we should not force `HostNameVerification` 

see:
https://github.com/dotCMS/core/blob/master/dotCMS/src/main/java/com/dotcms/content/elasticsearch/util/DotRestHighLevelClientProvider.java#L152

**Comment by dsilvam** [comment]
PR: https://github.com/dotCMS/core/pull/19243

**Comment by victoralfaro-dotcms** [comment]
Integrity Check data generated correctly in receiver cluster. Configures cluster consist in an Apache Load Balancer (mod_jk) sat before a two DotCMS node clusters sharing the same DB.
On receiver I've created a folder and a page to generate conflicts between sender and receivers. When running integrity check I got this from Apache log:

```
127.0.0.1 - - [01/Oct/2020:15:43:18 -0600] "POST /api/integrity/_generateintegritydata HTTP/1.1" 200 36
127.0.0.1 - - [01/Oct/2020:15:43:18 -0600] "POST /api/integrity/77ccb0b5-cd93-4437-9aab-1d6b54239524/status HTTP/1.1" 200 2515
```

This log from the sender host (8080):

```
01-Oct-2020 15:43:28.677 WARNING [url:GET//demo.dotcms.com/api/integrity/checkIntegrityProcessStatus/endPoint/cac41809-da38-43a6-95d8-0e8b11982167 | lang:1 | ip:0:0:0:0:0:0:0:1 | Admin:false | start:10-01-2020 03:22:03 CST  ref:http://localhost:8080/c/portal/layout?p_l_id=a8e430e3-8010-40cf-ade1-5978e61241a8&p_p_id=configuration&p_p_action=0&&dm_rlout=1&r=1601587329648&in_frame=true&frame=detailFrame&container=true&angularCurrentPortlet=configuration  ?dojo.preventCache=1601588608629] org.glassfish.jersey.servlet.WebComponent.filterFormParameters A servlet request to the URI http://localhost:8080/api/integrity/checkIntegrityProcessStatus/endPoint/cac41809-da38-43a6-95d8-0e8b11982167?dojo.preventCache=1601588608629 contains form parameters in the request body but the request body has been consumed by the servlet or a servlet filter accessing the request parameters. Only resource methods using @FormParam will work as expected. Resource methods consuming the request body by other means will not work as expected.
15:44:00.103  INFO  business.PublisherQueueJob - ===========================================================
15:44:00.103  INFO  business.PublisherQueueJob - For bundle '01EKGEKFRWEPVAWNQMSEWFHFYH':
15:44:00.103  INFO  business.PublisherQueueJob - -> Status             : PUBLISHING_BUNDLE [12]
15:44:00.103  INFO  business.PublisherQueueJob - ===========================================================
01-Oct-2020 15:44:11.093 WARNING [url:GET//demo.dotcms.com/api/integrity/getIntegrityResult/endPoint/cac41809-da38-43a6-95d8-0e8b11982167 | lang:1 | ip:0:0:0:0:0:0:0:1 | Admin:false | start:10-01-2020 03:22:01 CST  ref:http://localhost:8080/c/portal/layout?p_l_id=a8e430e3-8010-40cf-ade1-5978e61241a8&p_p_id=configuration&p_p_action=0&&dm_rlout=1&r=1601587329648&in_frame=true&frame=detailFrame&container=true&angularCurrentPortlet=configuration  ?dojo.preventCache=1601588651042] org.glassfish.jersey.servlet.WebComponent.filterFormParameters A servlet request to the URI http://localhost:8080/api/integrity/getIntegrityResult/endPoint/cac41809-da38-43a6-95d8-0e8b11982167?dojo.preventCache=1601588651042 contains form parameters in the request body but the request body has been consumed by the servlet or a servlet filter accessing the request parameters. Only resource methods using @FormParam will work as expected. Resource methods consuming the request body by other means will not work as expected.
01-Oct-2020 15:44:38.898 WARNING [url:GET//demo.dotcms.com/api/integrity/checkIntegrityProcessStatus/endPoint/cac41809-da38-43a6-95d8-0e8b11982167 | lang:1 | ip:0:0:0:0:0:0:0:1 | Admin:false | start:10-01-2020 03:22:14 CST  ref:http://localhost:8080/c/portal/layout?p_l_id=a8e430e3-8010-40cf-ade1-5978e61241a8&p_p_id=configuration&p_p_action=0&&dm_rlout=1&r=1601587329648&in_frame=true&frame=detailFrame&container=true&angularCurrentPortlet=configuration  ?dojo.preventCache=1601588678842] org.glassfish.jersey.servlet.WebComponent.filterFormParameters A servlet request to the URI http://localhost:8080/api/integrity/checkIntegrityProcessStatus/endPoint/cac41809-da38-43a6-95d8-0e8b11982167?dojo.preventCache=1601588678842 contains form parameters in the request body but the request body has been consumed by the servlet or a servlet filter accessing the request parameters. Only resource methods using @FormParam will work as expected. Resource methods consuming the request body by other means will not work as expected.
```

On the receiver nodes we see that one is taking charge of the actual data file generation and the other responds with the status of each:
![image](https://user-images.githubusercontent.com/64026161/94867424-7be34300-03fe-11eb-9e15-0c082e314329.png)

Conflicts were detected for both folder and page content:
![image](https://user-images.githubusercontent.com/64026161/94867520-9ddcc580-03fe-11eb-9eb3-3293187e1ecb.png)
![image](https://user-images.githubusercontent.com/64026161/94867530-a59c6a00-03fe-11eb-845a-0a443d835258.png)

**Comment by bryanboza** [comment]
Fixed, tested on release-20.10.1 // postgres // FF // docker
